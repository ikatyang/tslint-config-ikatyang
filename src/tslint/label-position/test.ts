// good

my_loop:
for (let i = 0; i < 10; i++) {
  break my_loop;
}

// bad

my_var:
var x = 1;
