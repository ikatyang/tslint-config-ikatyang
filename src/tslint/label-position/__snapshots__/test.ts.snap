// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

my_loop:
for (let i = 0; i < 10; i++) {
  break my_loop;
}

// bad

my_var:
~~~~~~  [unexpected label on statement]
var x = 1;

`;
